frontend/src/pages/Artwork/Artwork.tsx

```diff
@@ -26,17 +28,38 @@ import MetricBadge from "../../components/MetricBadge/MetricBadge";
 import Tag from "../../components/Tag/Tag";
 import ConnectionErrorMessage from "../../components/ConnectionErrorMessage/ConnectionErrorMessage";
 import { useParams, useHistory, Route, Switch } from "react-router-dom";
-import { useQuery, useMutation } from "@apollo/client";
 import Spinner from "../../components/Spinner/Spinner";
 import Drawer from "../../components/Drawer/Drawer";
 import { useState } from "react";
 import { AnimatePresence } from "framer-motion";
 import useProfileInfo from "../../hooks/useProfileInfo";
+import usePhotoLibrary from "../../hooks/usePhotoLibrary";
+
+const ADD_PHOTOS_MUTATION = gql`
+  mutation addPhotos(
+    $artworkId: ID!
+    $pictures_to_add: [String]
+  ) {
+    updateArtwork(
+      artworkData: {
+        id: $artworkId
+        pictures: $pictures_to_add
+      }
+    ) {
+      artwork {
+        id
+      }
+    }
+  }
+`;
```

Comment:
Can this be moved to the `gql.ts` file where all of the other mutations and queries are defined? This keeps the component file a little more concise and divides concerns.

Reply:
Sounds good, will do!